src/lib/MiradorViewer.js

```diff
@@ -42,7 +42,9 @@ class MiradorViewer {
    * Process config into actions
    */
   processConfig() {
-    const mergedConfig = deepmerge(settings, this.config);
+    /** merge type for arrays */
+    const overwriteMerge = (destinationArray, sourceArray, options) => sourceArray;
```

Comment:
This may be the only "controversial" thing?

Reply:
From our current settings, it looks like this would only  apply to motivations (where it makes sense) and windows (an empty array, so it doesn't matter).